src/Element.php

```diff
@@ -248,6 +250,22 @@ public function prop_set_selected(bool $selected):bool {
 		return $this->selected;
 	}
 
+	public function prop_get_form() {
+		if (\in_array($this->tagName, self::$formControlElements)) {
+			if ($this->tagName === "input" && $this->getAttribute("type") === "image")
+				return null;
+
+			if ($this->hasAttribute("form")) {
```

Comment:
Using the `form` attribute is new to me. Just a question: When an input element is placed outside of a form, and assigned the form using this attribute, is there anything else we need to unit test to make sure that the data is passed correctly to the server?

Reply:
Two things that I'm thinking of:

- "elements" property on form object must include those elements outside of the form but with form attribute.

- when element is child of form element but refers to other form with form attribute, it must be excluded from parent form "elements" collection.

Reply:
Interesting. Looks like there are more properties that can be implemented. I'll start a new issue to track things like this, and we can work towards implementing all of these extra DOM level 4 properties.

New issue to track properties: #168